packages/iris-grid/src/asserts.tsx

```diff
@@ -1,15 +1,5 @@
-export function assertNotNull<T>(value: T | null): asserts value is T {
-  if (value == null) throw new Error('Value is null');
-}
-
-export function assertNotUndefined<T>(
-  value: T | undefined
-): asserts value is T {
-  if (value === undefined) throw new Error('Value is undefined');
-}
-export function assertNotNullNorUndefined<T>(
+export default function assertNotNull<T>(
```

Comment:
Should either name the file `assertNotNull.ts` or change the export to `export function assertNotNull` (may need to suppress a warning about preferring default with one export).

It doesn't make sense for the file to be called `asserts` and only have one function exported as default.

Also - it makes sense to move this to `@deephaven/utils` package. I think it will come in handy outside of iris-grid.